Index: xkb-data/rules/base.o_s.part
===================================================================
--- xkb-data.orig/rules/base.o_s.part
+++ xkb-data/rules/base.o_s.part
@@ -73,6 +73,7 @@
   keypad:future         =	+keypad(future)
   keypad:future_wang    =	+keypad(future_wang)
   keypad:phone_hex      =	+keypad(phone_hex)
+  keypad:comma          =	+keypad(comma)
   nbsp:none             =	+nbsp(none)
   nbsp:level2           =	+nbsp(level2)
   nbsp:level3           =	+nbsp(level3)
Index: xkb-data/rules/base.xml.in
===================================================================
--- xkb-data.orig/rules/base.xml.in
+++ xkb-data/rules/base.xml.in
@@ -4311,6 +4311,12 @@
       </option>
       <option>
         <configItem>
+          <name>keypad:comma</name>
+          <_description>Force standard comma keypad</_description>
+       </configItem>
+      </option>
+      <option>
+        <configItem>
           <name>keypad:oss</name>
           <_description>Keypad with unicode additions (arrows and math operators)</_description>
        </configItem>
