Index: xkb-data/rules/base.o_s.part
===================================================================
--- xkb-data.orig/rules/base.o_s.part	2008-05-28 14:12:01.000000000 +0200
+++ xkb-data/rules/base.o_s.part	2008-05-28 14:12:01.000000000 +0200
@@ -70,7 +70,8 @@
   keypad:legacy_wang    =	+keypad(legacy_wang)
   keypad:oss_wang       =	+keypad(oss_wang)
   keypad:future_wang    =	+keypad(future_wang)
   keypad:phone_hex      =	+keypad(phone_hex)
+  keypad:comma          =	+keypad(comma)
   nbsp:none             =	+nbsp(none)
   nbsp:level2           =	+nbsp(level2)
   nbsp:level3           =	+nbsp(level3)
Index: xkb-data/rules/base.xml.in
===================================================================
--- xkb-data.orig/rules/base.xml.in	2008-05-28 14:12:01.000000000 +0200
+++ xkb-data/rules/base.xml.in	2008-05-28 14:12:01.000000000 +0200
@@ -4233,6 +4233,12 @@
       </option>
       <option>
         <configItem>
+          <name>keypad:comma</name>
+          <_description>Force standard comma keypad</_description>
+       </configItem>
+      </option>
+      <option>
+        <configItem>
           <name>keypad:oss</name>
           <_description>Keypad with unicode additions (arrows and math operators)</_description>
        </configItem>
