Index: xkeyboard-config/rules/base.o_s.part
===================================================================
--- xkeyboard-config.orig/rules/base.o_s.part	2009-01-09 02:43:51.424898319 +0300
+++ xkeyboard-config/rules/base.o_s.part	2009-01-09 02:44:00.876897709 +0300
@@ -78,6 +78,7 @@
   keypad:future_wang    =	+keypad(future_wang)
   keypad:hex            =	+keypad(ops)+keypad(hex)
   keypad:atm            =	+keypad(ops)+keypad(hex)+keypad(atm)  
+  keypad:comma          =	+keypad(comma)
   nbsp:none             =	+nbsp(none)
   nbsp:level2           =	+nbsp(level2)
   nbsp:level3           =	+nbsp(level3)
Index: xkeyboard-config/rules/base.xml.in
===================================================================
--- xkeyboard-config.orig/rules/base.xml.in	2009-01-09 02:43:51.424898319 +0300
+++ xkeyboard-config/rules/base.xml.in	2009-01-09 02:44:00.876897709 +0300
@@ -4442,6 +4442,12 @@
       </option>
       <option>
         <configItem>
+          <name>keypad:comma</name>
+          <_description>Force standard comma keypad</_description>
+       </configItem>
+      </option>
+      <option>
+        <configItem>
           <name>keypad:oss</name>
           <_description>Keypad with unicode additions (arrows and math operators)</_description>
        </configItem>
