--- xkeyboard-config-1.3/symbols/inet	2008-12-02 11:30:26.000000000 -0600
+++ xkeyboard-config-1.3/symbols/inet.new	2008-12-02 14:43:19.000000000 -0600
@@ -158,9 +158,9 @@
     key <I241>   {      [ XF86MailForward       ]       };
     key <I242>   {      [ XF86Save              ]       };
     key <I243>   {      [ XF86Documents         ]       };
-//  key <I244>   {      [ ]       }; // KEY_BATTERY
-//  key <I245>   {      [ ]       }; // KEY_BLUETOOTH
-//  key <I246>   {      [ ]       }; // KEY_WLAN
+    key <I244>   {      [ XF86Battery           ]       };
+    key <I245>   {      [ XF86Bluetooth         ]       };
+    key <I246>   {      [ XF86WLAN              ]       };
 //  key <I249>   {      [ ]       }; // KEY_VIDEO_NEXT  --  drive next video source
 //  key <I250>   {      [ ]       }; // KEY_VIDEO_PREV  --  drive previous video source
 //  key <I251>   {      [ ]       }; // KEY_BRIGHTNESS_CYCLE  --  bright up, max++ == min
