Index: xkbdata-X11R7.0-1.0.1/rules/xorg.xml
===================================================================
--- xkbdata-X11R7.0-1.0.1.orig/rules/xorg.xml	2006-01-26 21:46:48.000000000 -0500
+++ xkbdata-X11R7.0-1.0.1/rules/xorg.xml	2006-02-26 18:28:52.000000000 -0500
@@ -619,6 +619,18 @@
     </model>
     <model>
       <configItem>
+        <name>ibm_space_saver</name>
+        <description>IBM Space Saver</description>
+      </configItem>
+    </model>
+    <model>
+      <configItem>
+        <name>cherryblueb</name>
+        <description>Cherry CyMotion Master XPress</description>
+      </configItem>
+    </model>
+    <model>
+      <configItem>
         <name>cherrybluea</name>
         <description>Cherry Blue Line CyBo@rd (alternate option)</description>
         <description xml:lang="af">Cherry Blue Line CyBo@rd (alternatief)</description>
Index: xkbdata-X11R7.0-1.0.1/symbols/inet
===================================================================
--- xkbdata-X11R7.0-1.0.1.orig/symbols/inet	2006-01-26 21:46:48.000000000 -0500
+++ xkbdata-X11R7.0-1.0.1/symbols/inet	2006-02-26 18:28:52.000000000 -0500
@@ -353,6 +353,48 @@
 
 // Cherry Blue Line CyBo@rd (alternate option)
 partial alphanumeric_keys
+xkb_symbols "cherryblueb" {
+    name[Group1]= "Cherry CyMotion Master XPress";
+
+    key <I0B>	{	[ XF86ScrollUp		]	};
+    key <I12>   {       [ XF86ScrollDown        ]       };
+    key <I17>   {       [ XF86Cut               ]       };
+    key <I0A>   {       [ XF86Copy              ]       };
+    key <I18>   {       [ XF86Paste             ]       };
+    key <I71>   {       [ XF86Book              ]       };
+    key <I2C>   {       [ XF86Book              ]       };
+    key <I72>   {       [ XF86Book              ]       };
+    key <I07>   {       [ XF86Forward           ]       };
+    key <I08>   {       [ XF86Back              ]       };
+    key <I2E>   {       [ XF86AudioLowerVolume  ]       };
+    key <I30>   {       [ XF86AudioRaiseVolume  ]       };
+    key <I20>   {       [ XF86AudioMute         ]       };
+    key <I6D>   {       [ XF86AudioMedia        ]       };
+    key <I67>   {       [ XF86AudioRewind       ]       };
+    key <I10>   {       [ XF86Back              ]       };
+    key <I19>   {       [ XF86Forward           ]       };
+    key <I70>   {       [ XF86Eject             ]       };
+    key <I32>   {       [ XF86Finance           ]       };
+    key <I6A>   {       [ XF86Back              ]       };
+    key <I69>   {       [ XF86Forward           ]       };
+    key <I68>   {       [ XF86Stop              ]       };
+    key <FK16>  {       [ XF86Reload            ]       };
+    key <I65>   {       [ XF86Search            ]       };
+    key <I66>   {       [ XF86MyComputer        ]       };
+    key <I02>   {       [ XF86HomePage          ]       };
+    key <I6C>   {       [ XF86Mail              ]       };
+    key <PRSC>  {       [ XF86Terminal          ]       };
+    key <I21>   {       [ XF86Calculator        ]       };
+    key <I1E>   {       [ XF86AudioRaiseVolume  ]       };
+    key <I26>   {       [ XF86AudioMute         ]       };
+    key <I25>   {       [ XF86AudioLowerVolume  ]       };
+    key <XFER>  {       [ XF86Go                ]       };
+    key <I24>   {       [ XF86AudioStop                 ] };
+    key <I22>   {       [ XF86AudioPlay, XF86AudioPause ] };
+    key <I63>   {       [ XF86Standby                   ] };
+};
+
+partial alphanumeric_keys
 xkb_symbols "cherrybluea" {
     key <I6A>	{	[ XF86Back		]	};
     key <I69>	{	[ XF86Forward		]	};
