Index: xkb-data/keycodes/evdev
===================================================================
--- xkb-data.orig/keycodes/evdev	2008-09-16 13:15:01.000000000 +0300
+++ xkb-data/keycodes/evdev	2008-09-17 12:23:29.000000000 +0300
@@ -147,14 +147,14 @@
 
 	<HZTG> =  93;	// Hankaku/Zenkakau toggle
 	<HKTG> = 101;	// Hiragana/Katakana toggle
-	<AB11> = 211;	// backslash/underscore
+	<AB11> =  97;	// backslash/underscore
 	<HENK> = 100;	// Henkan
 	<MUHE> = 102;	// Muhenkan
 	<AE13> = 132;	// Yen
 	<KATA> =  98;	// Katakana
 	<HIRA> =  99;	// Hiragana
 	<JPCM> = 103;	// KPJPComma
-	<RO>   =  97;	// Romaji
+	//<RO>   =  97;	// Romaji
 
 	// Keys that are generated on Korean keyboards
 
@@ -235,7 +235,7 @@
 	<I208> = 208;	// #define KEY_PLAYCD              200
 	<I209> = 209;	// #define KEY_PAUSECD             201
 	<I210> = 210;	// #define KEY_PROG3               202
-	//<I211> = 211;	// #define KEY_PROG4               203 conflicts with AB11
+	<I211> = 211;	// #define KEY_PROG4               203
 	<I213> = 213;	// #define KEY_SUSPEND             205
 	<I214> = 214;	// #define KEY_CLOSE               206
 	<I215> = 215;	// #define KEY_PLAY                207
@@ -304,6 +304,8 @@
 // For japanese 106 keyboard.
 xkb_keycodes "jp106" {
     include "evdev(evdev)"
+    <HZTG> = 49;
+    <AC12> = 51;
 };
 
 // For brazilian ABNT2 keyboard.
Index: xkb-data/symbols/inet
===================================================================
--- xkb-data.orig/symbols/inet	2008-09-17 12:23:52.000000000 +0300
+++ xkb-data/symbols/inet	2008-09-17 12:24:11.000000000 +0300
@@ -126,7 +126,7 @@
     key <I208>   {      [ XF86AudioPlay         ]       };
     key <I209>   {      [ XF86AudioPause        ]       };
     key <I210>   {      [ XF86Launch3           ]       };
-    key <AB11>   {      [ XF86Launch4           ]       };
+    key <I211>   {      [ XF86Launch4           ]       };
     key <I213>   {      [ XF86Standby           ]       };
     key <I214>   {      [ XF86Close             ]       };
     key <I215>   {      [ XF86AudioPlay         ]       };
