Let 'make clean' remove generated files.

Not submitted upstream.

Index: xkb-data/rules/compat/Makefile.am
===================================================================
--- xkb-data.orig/rules/compat/Makefile.am
+++ xkb-data/rules/compat/Makefile.am
@@ -7,15 +7,17 @@
 
 if USE_COMPAT_RULES
 
-noinst_SCRIPTS=base.l2_s.part base.l3_s.part base.l4_s.part \
+gen_parts=base.l2_s.part base.l3_s.part base.l4_s.part \
 base.l2v2_s.part base.l3v3_s.part base.l4v4_s.part \
 base.ml_s.part base.ml1_s.part \
 base.mlv_s.part base.ml1v1_s.part \
-base.ml_s.part $(dist_parts)
+base.ml_s.part
+noinst_SCRIPTS=$(gen_parts) $(dist_parts)
 
 else
 
 noinst_SCRIPTS=
+gen_parts=
 
 endif
 
@@ -50,3 +52,4 @@
 	sh $<
 
 EXTRA_DIST=base.lists.part $(transform_files) $(transform_scripts) $(dist_parts)
+CLEANFILES=$(gen_parts)
Index: xkb-data/rules/compat/Makefile.in
===================================================================
--- xkb-data.orig/rules/compat/Makefile.in
+++ xkb-data/rules/compat/Makefile.in
@@ -148,14 +148,17 @@
 transform_files = layoutRename.lst variantRename.lst
 transform_scripts = ln_s.sh lnv_s.sh ml_s.sh mlv_s.sh ml1_s.sh ml1v1_s.sh 
 dist_parts = base.o_s.part
-@USE_COMPAT_RULES_FALSE@noinst_SCRIPTS = 
-@USE_COMPAT_RULES_TRUE@noinst_SCRIPTS = base.l2_s.part base.l3_s.part base.l4_s.part \
+@USE_COMPAT_RULES_FALSE@gen_parts = 
+@USE_COMPAT_RULES_TRUE@gen_parts = base.l2_s.part base.l3_s.part base.l4_s.part \
 @USE_COMPAT_RULES_TRUE@base.l2v2_s.part base.l3v3_s.part base.l4v4_s.part \
 @USE_COMPAT_RULES_TRUE@base.ml_s.part base.ml1_s.part \
 @USE_COMPAT_RULES_TRUE@base.mlv_s.part base.ml1v1_s.part \
-@USE_COMPAT_RULES_TRUE@base.ml_s.part $(dist_parts)
+@USE_COMPAT_RULES_TRUE@base.ml_s.part
 
+@USE_COMPAT_RULES_FALSE@noinst_SCRIPTS = 
+@USE_COMPAT_RULES_TRUE@noinst_SCRIPTS = $(gen_parts) $(dist_parts)
 EXTRA_DIST = base.lists.part $(transform_files) $(transform_scripts) $(dist_parts)
+CLEANFILES = $(gen_parts)
 all: all-am
 
 .SUFFIXES:
@@ -244,6 +247,7 @@
 mostlyclean-generic:
 
 clean-generic:
+	-test -z "$(CLEANFILES)" || rm -f $(CLEANFILES)
 
 distclean-generic:
 	-test -z "$(CONFIG_CLEAN_FILES)" || rm -f $(CONFIG_CLEAN_FILES)
