From 8e253e1bedfaf4beba3763c1e6b9ef4a706487a3 Mon Sep 17 00:00:00 2001
From: Sergey V. Udaltsov <svu@localhost.localdomain>
Date: Sun, 3 Aug 2008 22:30:33 +0100
Subject: [PATCH] fixed compat rules for Romania, b.fd.o#16872

---
 rules/compat/variantRename.lst |    7 ++++---
 2 files changed, 9 insertions(+), 3 deletions(-)

Index: xkb-data/rules/compat/variantRename.lst
===================================================================
--- xkb-data.orig/rules/compat/variantRename.lst
+++ xkb-data/rules/compat/variantRename.lst
@@ -25,9 +25,10 @@
 ogham	is434laptop	ie	ogam_is434
 ori	basic		in	ori
 ro	de		ro	winkeys
-ro	us		ro	std_comma
-ro	std		ro	std_cedilla
-ro	academic	ro	std_comma
+ro	us		ro	std
+ro	academic	ro	std
+ro	std_comma	ro	std
+ro	comma		ro	basic	
 ru	os		ru	os_legacy
 sapmi	basic		no	smi
 sapmi	nodeadkeys	no	smi_nodeadkeys
