Index: xkeyboard-config-0.8/symbols/macintosh_vndr/gb
===================================================================
--- xkeyboard-config-0.8.orig/symbols/macintosh_vndr/gb
+++ xkeyboard-config-0.8/symbols/macintosh_vndr/gb
@@ -10,6 +10,7 @@ xkb_symbols "basic" {
 
     name[Group1]= "United Kingdom - Mac";
 
+    key <AE02> {	[               2,              at,         EuroSign	]	};
     key <AE03> {	[               3,        sterling,       numbersign	]	};
 
     // End alphanumeric section
Index: xkeyboard-config-0.8/symbols/gb
===================================================================
--- xkeyboard-config-0.8.orig/symbols/gb
+++ xkeyboard-config-0.8/symbols/gb
@@ -89,6 +89,7 @@ xkb_symbols "mac" {
 
     name[Group1]= "United Kingdom - Macintosh";
 
+    key <AE02> {	[               2,              at,         EuroSign	]	};
     key <AE03> {	[               3,        sterling,       numbersign	]	};
 
     // End alphanumeric section
